scripts/copy-component

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env bash
+
+usage="
+$0: Copy Component Definition from within the templates directory or from the network
+
+Usage:
+    $0 -h
+    $0 -n COMPONENT_NAME [-u COMPONENT_URL]
+
+Options:
+-h: show help and exit
+-n: Local Component Name
```

Comment:
Since README line 61 reads

> To import a component that is available from a URL: `copy-component -n COMPONENT_NAME -u COMPONENT_URL`

I think this should be:

`-n: Component Name` (and not "Local Component")